os/fs/smartfs/smartfs_utils.c

```diff
@@ -858,9 +858,9 @@ int smartfs_mount(struct smartfs_mountpt_s *fs, bool writeable)
 
 	if (nextfs == NULL) {
 #ifdef CONFIG_SMARTFS_DYNAMIC_HEADER
-		fs->fs_chunk_buffer = (char *)kmm_malloc(fs->fs_llformat.availbytes);
+		fs->fs_chunk_buffer = (char *)kmm_malloc(AVAIL_BYTES(fs));
```

Comment:
`fs_chunk_buffer` is uint8_t type, so I think we'd better make type casting as uint8_t.

Reply:
I'll keep this in mind for checking, thank you.
However, the current commit is only for replacing the long string parameter with a macro.